app/views/gigs/manage.html.slim

```diff
@@ -0,0 +1,20 @@
+= provide :title, "Manage Gigs"
+section.main-content
+  .row
+    .two.columns.offset-by-one.schedule
+      article
+        = link_to "Add gig", new_gig_path
+
+  - Gig.all_ascending.each do |gig|
+    -if gig.date + gig.days >= Date.today
```

Comment:
I'd probably tend to do a `select` with a block on the `all_ascending` collection to filter to just those with dates high enough...and the inverse below.